R/AddModules.R

```diff
@@ -4,7 +4,6 @@
 #' @description { This function adds a module to a shiny app created by BaSS.  }
 #'
 #' @param vModuleIDs IDs of the shiny module to be added to the application. The source code for each module should be saved in strModuleDirectory with files named "mod_{strModuleID}UI.R" and mod_{strModuleID}Server.R". If desired, submodules may also be included using the following naming convention "mod_{strModuleID}_XXX.R".
-#' @param strModuleID alias for vModuleIDs. Items are merged if both are provided.
 #' @param strPackageDirectory The directory where the BaSS shiny app located.
 #' @param strModuleDirectory Location of the Module to be added ot the app. Defaults to inst/_shared/modules.
 #' @param strType type of application - valid options are "package" and "standalone"
```

Comment:
I tried calling this with bDashboard = FALSE and it still adds the module to the dasboad menu items on the left.  If I call it with FALSE it should be the UI and SERVER module but not add it to the app and then the user adds the module where it is needed, correct?

Reply:
Moved to a new issue.